content/manuals/docker-hub/repos/manage/builds/setup.md

```diff
@@ -0,0 +1,301 @@
+
+2. Select the **Builds** tab.
+
+3. Select either GitHub or Bitbucket to connect where the image's source code is stored.
+
+   > [!NOTE]
+   >
+   > You may be redirected to the settings page to [link](link-source.md) the
+   > code repository service. Otherwise, if you are editing the build settings
+   > for an existing automated build, select **Configure automated builds**.
+
+4. Select the **source repository** to build the Docker images from.
+
+   > [!NOTE]
+   >
+   > You might need to specify an organization or user from
+   > the source code provider. Once you select a user, source code
+   > repositories appear in the **Select repository** drop-down list.
+
+5. Optional: Enable [autotests](automated-testing.md#enable-automated-tests-on-a-repository).
+
+6. Review the default **Build Rules**.
+
+    Build rules control what Docker Hub builds into images from the contents
+    of the source code repository, and how the resulting images are tagged
+    within the Docker repository.
+
+    A default build rule is set up for you, which you can edit or delete. This
+    default rule sets builds from the `Branch` in your source code repository called
+    `master` or `main`, and creates a Docker image tagged with `latest`. For more information, see [set up build rules](#set-up-build-rules)
+
+7. Optional: Select the **plus** icon to add and [configure more build rules](#set-up-build-rules).
+
+8. For each branch or tag, enable or disable the **Autobuild** toggle.
+
+    Only branches or tags with autobuild enabled are built, tested, and have
+    the resulting image pushed to the repository. Branches with autobuild
+    disabled are built for test purposes (if enabled at the repository
+    level), but the built Docker image isn't pushed to the repository.
+
+9. For each branch or tag, enable or disable the **Build Caching** toggle.
+
+    [Build caching](/manuals/build/building/best-practices.md#leverage-build-cache)
+    can save time if you are building a large image frequently or have
+    many dependencies. Leave the build caching disabled to
+    make sure all of your dependencies are resolved at build time, or if
+    you have a large layer that's quicker to build locally.
+
+10. Select **Save** to save the settings, or select **Save and build** to save and
+   run an initial test.
+
+    > [!NOTE]
+    >
+    > A webhook is automatically added to your source code repository to notify
+    > Docker Hub on every push. Only pushes to branches that are listed as the
+    > source for one or more tags, trigger a build.
+
+### Set up build rules
+
+By default when you set up automated builds, a basic build rule is created for you.
+This default rule watches for changes to the `master` or `main` branch in your source code
+repository, and builds the `master` or `main` branch into a Docker image tagged with
+`latest`.
+
+In the **Build Rules** section, enter one or more sources to build.
```

Comment:
is this still in Docker Hub > Build section?

Reply:
Yes. "Build Rules" is a section under the "Builds" tab. 
![image](https://github.com/user-attachments/assets/ea23d21b-c9c0-41d0-94cf-540766291b44)